test/test.js

```diff
@@ -68,6 +82,10 @@ describe('chai-immutable (' + typeEnv + ')', function () {
       it('should fail using `not` given an empty collection', function () {
         fail(function () { expect(new List()).to.not.be.empty; });
       });
+
+      it('should work if using different copies of Immtuable', function () {
+        expect(clonedImmutable.List()).to.be.empty;
```

Comment:
By the way, why all these tests? I understand when asserting that 2 lists from 2 different copies of Immutable.js are identical, but here we are testing the same thing than [this](https://github.com/astorije/chai-immutable/pull/36/files#diff-c1129c8b045390789fa8ff62f2c6b4a9R65). Unless that's the point, to test that the cloned module works fine by itself?